SimulationOutputReport/tests/output_report_test.rb

```diff
@@ -26,6 +26,7 @@ class SimulationOutputReportTest < MiniTest::Test
     "Electricity: Lighting Exterior (MBtu)",
     "Electricity: Mech Vent (MBtu)",
     "Electricity: Whole House Fan (MBtu)",
+    "Electricity: Dehumidifier (MBtu)",
```

Comment:
Eventually it should also be included in:
- TimeseriesColsEndUses: "Electricity: Dehumidifier"
- AnnualRows: "Component Load: Heating: Dehumidifier (MBtu)"
- AnnualRows: "Component Load: Cooling: Dehumidifier (MBtu)"
- TimeseriesColsComponentLoads: "Component Load: Heating: Dehumidifier"
- TimeseriesColsComponentLoads: "Component Load: Cooling: Dehumidifier"

Reply:
In component load program, we're including dehumidifier into internal gains. Do you mean separating it out here? Or both places?

Reply:
Ah gotcha, I didn't realize that. In that case, only TimeseriesColsEndUses above needs to be addressed.

Reply:
Do you mean the TimeseriesColsEndUses is also expected in hpxmls where no dehumidifier exists? Now dehumidifier column is added into annual results even if there's no dehumidifier (because it's populated as 0 when is nil), but not into timeseries. Should I change it to :
1. No humidifier hpxml: no dehumidifier annual, no dehumidifier timeseries;
2. Has humidifier hpxml: have dehumidifier annual, have dehumidifier timeseries;

OR:
1. No humidifier hpxml: have dehumidifier annual, have dehumidifier timeseries;
2. Has humidifier hpxml: have dehumidifier annual, have dehumidifier timeseries;

Reply:
The latter. I'm trying to keep the list of end uses static across all buildings, as well as keep the list identical between annual and timeseries. Note that "Electricity: Whole House Fan" is in both lists even though the test file has no whole house fan.

(I hope you meant dehumidifier above and not humidifier! 😆)